lib/keeper/configuration.rb

```diff
@@ -6,14 +6,25 @@ class Configuration < OpenStruct
       expiry:           24.hours,
       issuer:           'api.example.com',
       audience:         'example.com',
-      redis_connection: nil
-    }
+      redis_connection: nil,
+      version:          nil
+    }.freeze
 
     # Creates a new Configuration from the passed in parameters
     # @param params [Hash] configuration options
     # @return [Configuration]
     def initialize(params = {})
       super(DEFAULTS.merge(params))
     end
+
+    # @!visibility private
+    def base_claims
+      {
+        iss: Keeper.configuration.issuer,               # issuer
+        aud: Keeper.configuration.audience,             # audience
+        exp: Keeper.configuration.expiry.from_now.to_i, # expiration time
+        ver: Keeper.configuration.version               # Version
```

Comment:
This functionality is integral to being able to handle the rolling deployment functionality which we implicitly accept responsibility with the hard and soft invalidation layers. The version usage itself is completely unopinionated, as it only cares about comparing the values.